<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QWidget" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="codeSpace">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>661</width>
     <height>611</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="save_py_select">
   <property name="geometry">
    <rect>
     <x>685</x>
     <y>470</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>.py</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="save_mdw_select">
   <property name="geometry">
    <rect>
     <x>775</x>
     <y>470</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>.mdw</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>450</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>570</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>10</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Ejecutar</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>-10</y>
     <width>271</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>MDW  NoCodePython</string>
   </property>
  </widget>
  <widget class="QPushButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ayuda ðŸ†˜</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>60</y>
     <width>281</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="commandLine">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>360</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>360</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>120</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Limpiar</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>190</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="languajeComboBox">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>260</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ðŸ‡ªðŸ‡¸</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ðŸ‡¬ðŸ‡§</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ðŸ‡«ðŸ‡·</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
